### PR TITLE
Implemented backgroundColor method for overriding the game background

### PR DESCRIPTION
**Pull request by wolfenrain**
Implemented backgroundColor method for overriding the game background

### CODE REVIEW

**Comment by erickzanardo** [comment]
Awesome, looking good, just two missing things:
 - You need to rebase your PR to the develop branch
 - Add an entry to the CHANGELOG.md with your change, be sure to add a (thanks @wolfenrain) on the end of it

File: lib/game.dart [hunk 1]
```diff
@@ -23,6 +23,10 @@ abstract class Game {
   // Widget Builder for this Game
   final builder = WidgetBuilder();
 
+  /// Returns the game background color.
+  /// By default it will return a black color.
+  Color backgroundColor() => const Color(0xFF000000);
```
Comment (luanpotter) [inline lib/game.dart]: Please add to the comments explaining that this value cannot be changed at runtime.

Comment (wolfenrain) [reply lib/game.dart]: Agreed, that can clear up some confusion beforehand. Consider it done

**Comment by erickzanardo** [comment]
Thanks @wolfenrain

**Pull request merged**